modules/ocf/manifests/motd.pp

```diff
@@ -1,12 +1,12 @@
-class ocf::motd {
-  $is_virtual = str2bool($::is_virtual)
-
-  if tagged('ocf_ssh') or tagged('ocf_apphost') {
+class ocf::motd($nfs_motd = false) {
```

Comment:
Alright, sounds like a good idea to me, that seems to cover all the times we'd want to potentially show that motd, and I'm sure having a public attribute would be useful elsewhere.